src/cli/commands/telemetry/telemetry.ts

```diff
@@ -0,0 +1,57 @@
+import { logger } from "../../../core";
+import path from "path";
+import dotenv from "dotenv";
+import { ENV_FILENAME } from "../../../core/constants";
+import { updateGitIgnore } from "../../../core/git";
+import { existsSync, promises as fsPromises } from "fs";
+const { readFile, writeFile } = fsPromises;
+
+export async function telemetry(options: SWACLIConfig) {
+  let { disable, enable, status } = options;
+  const envFile = path.join(process.cwd(), ENV_FILENAME);
```

Comment:
Check if we can refactor and put the ENV_FILENAME reading and writing to a separate helper file and use it in login/telemetry flow

Reply:
added a method to read file